test/athens/common_events/left_sidebar_test.cljc

```diff
@@ -1,212 +0,0 @@
-(ns athens.common-events.left-sidebar-test
```

Comment:
What happened here?

Reply:
I think we refactored these events into atomic events, disabled the test here, and never deleted it. I only found it because I removed the `.resolve` ns due to removing undo.